src/ApimEventProcessor/Helpers/ParamConfig.cs

```diff
@@ -48,11 +53,13 @@ public static class AppExecuteParams
     public static class RunParams
     {
         // Frequency at which events are checkpointed to Azure Storage.
-        public const int CHECKPOINT_MINIMUM_INTERVAL_MINUTES = 5;
+        public const int CHECKPOINT_MINIMUM_INTERVAL_MINUTES = 1;
         
```

Comment:
Did we want to change it to 1 minutes?

Reply:
yes.. to improve perf of re-starting web job.. so it doesnt process what it doesnt need to.